components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,46 @@
+import Tabs, { TabsProps } from '@mui/material/Tabs'
+
+type BreadcrumbsProps = TabsProps
+
+/**
+ * `Breadcrumbs`/`Breadcrumb` element, similar to `Tabs`/`Tab` element. Usage is based on the
+ * [MUI Tabs](https://mui.com/material-ui/react-tabs/#basic-tabs) example.
+ *
+ * @example
+ * ```ts
+ * const Example = () => {
+ *   const [value, setValue] = useState(0)
+ *   return (
+ *     <div>
+ *       <Box>
+ *         <Breadcrumbs value={value} onChange={(_, v) => setValue(v)}>
+ *           <Breadcrumb label="Select Campaign" />
+ *           <Breadcrumb label="Select Payment" />
+ *           <Breadcrumb label="Donation Amount" />
+ *           <Breadcrumb label="My Details" />
+ *           <Breadcrumb label="Submit" />
+ *         </Breadcrumbs>
+ *       </Box>
+ *       <div hidden={value != 0}>Item One</div>
+ *       <div hidden={value != 1}>Item Two</div>
+ *       <div hidden={value != 2}>Item Three</div>
+ *       <div hidden={value != 3}>Item Four</div>
+ *       <div hidden={value != 4}>Item Five</div>
+ *     </div>
+ *   )
+ * }
+ * ```
+ */
```

Comment:
is this like a special documentation function?

Reply:
yep! I'll add it in to notion docs soon

![image](https://user-images.githubusercontent.com/12296254/200290453-db188561-79f6-4cc9-a5bc-562041269585.png)